scripts/prepare.ts

```diff
@@ -73,19 +76,27 @@ for (const [pn, artifact] of Object.entries(paymentNetworks)) {
 
 for (const network of networks) {
   const dataSources: DataSource[] = [];
+  console.log(`parsing network ${network}`);
 
   Object.entries(paymentNetworks).forEach(([pn, artifact]) => {
+    let graphEntities: string[];
+    if (pn === "ERC20EscrowToPay") {
+      graphEntities = ["Payment", "Escrow", "EscrowEvent"];
+    } else {
+      graphEntities = ["Payment"];
+    }
+
```

Comment:
Now entities are dynamically inserted in manifests. 
If a network has Escrow deployed it will contain all three entities, otherwise it only has Payment like before.